docs/framework/configure-apps/file-schema/wcf/authentication-of-clientcertificate-element.md

```diff
@@ -29,6 +29,7 @@ Specifies authentication behaviors for client certificates used by a service.
 ## Syntax  
   
 ```  
+  
```

Comment:
this change showing up here tells me you were no longer working with the latest from our master branch. do you know how to rebase so you don't reintroduce bugs we already fixed? i can send instructions offline if you don't.